src/shipit_code_coverage/shipit_code_coverage/artifacts.py

```diff
@@ -66,7 +68,7 @@ def download(self, test_task):
             logger.info('%s artifact downloaded' % artifact_path)
 
     def download_all(self):
-        mkdir('ccov-artifacts')
+        mkdir(self.parent_dir)
```

Comment:
Why not use [makedirs](https://docs.python.org/3/library/os.html#os.makedirs) here ?

Reply:
I didn't know there was a parameter to ignore if a directory already existed (it looks like it was added in Python 3.2).
I will file a good first issue to remove mkdir and use makedirs.